synapse/models/person.py

```diff
@@ -56,16 +119,23 @@ def getBaseModels():
                 ('ps:name',
                  {'ctor': 'synapse.models.person.Name', 'ex': 'smith,bob', 'doc': 'A last,first person full name'}),
                 ('ps:person',
-                 {'subof': 'guid', 'alias': 'ps:person:guidname', 'doc': 'A GUID for a person or suspected person'}),
+                 {'subof': 'guid', 'alias': 'ps:person:guidname', 'doc': 'A GUID for a person'}),
+                ('ps:persona',
+                 {'subof': 'guid', 'alias': 'ps:persona:guidname', 'doc': 'A GUID for a suspected person'}),
 
                 ('ps:contact', {'subof': 'guid', 'doc': 'A GUID for a contact info record'}),
 
-                ('ps:hasuser', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|user,inet:user'}),
-                ('ps:hashost', {'subof': 'comp', 'fields': 'person=ps:person,host=it:host'}),
-                ('ps:hasalias', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|alias,ps:name'}),
-                ('ps:hasphone', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|phone,tel:phone'}),
-                ('ps:hasemail', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|email,inet:email'}),
-                ('ps:haswebacct', {'subof': 'sepr', 'sep': '/', 'fields': 'person,ps:person|web:acct,inet:web:acct'}),
+                ('ps:has', {
+                    'subof': 'xref',
+                    'source': 'person,ps:person',
+                    'doc': 'A person owns, controls, or has exclusive use of an object or resource,'
+                        'potentially during a specific period of time.'}),
+
+                ('ps:persona:has', {
```

Comment:
@invisig0th @therealsilence - do we want this to be `ps:persona:has` or something else? I think I will probably need to modify #683 to convert `ps:has` to `ps:person:has` before merging that PR.